src/ann_solo/reader.py

```diff
@@ -1,11 +1,18 @@
+import io
+import mmap
+from multiprocessing import cpu_count
 import collections
 import logging
-import os
 import pickle
 from functools import lru_cache
+import os
```

Comment:
**todo:** Order imports using `import ...` before `from ... import ...` and alphabetically within each block.

Reply:
This was already fixed. Maybe you can check the last push in the current pull.